### ISSUE TITLE
iOS Build Profile `simulator` key is missing from Schema validator

### ISSUE THREAD

**Issue by sallar** [expo/eas-cli#545]
Having `simulator: true` in the new build profile format introduced in #537 is missing `simulator` key in the Joi schema:
https://github.com/expo/eas-cli/blob/main/packages/eas-json/src/EasJsonSchema.ts#L47

So running `eas build --profile development` fails like so:

```sh
$ eas build --profile development --platform ios
    Error: eas.json is not valid [ValidationError: "build.development.ios.simulator" is not allowed]
```

This is breaking the current development workflow after the migration to the new version.
I have created a PR to fix this issue: https://github.com/expo/eas-cli/pull/546

**Comment by wkozyra95** [comment]
fix published in eas-cli 0.22.4

**Comment by sallar** [comment]
Thanks @wkozyra95 confirmed that it works.
